### PR TITLE
Adds confirm_payment endpoint for manual confirmation

### PR DESCRIPTION
**Pull request by csabol-stripe**
Fixes manual confirmation confirm to include redirect URL

### CODE REVIEW

File: web.rb [hunk 1]
```diff
@@ -166,15 +186,18 @@ def create_payment_intent(amount, source_id, payment_method_id, customer_id = ni
...
+    :return_url => return_url,
+    :confirm => return_url != nil,
+    :confirmation_method => return_url == nil ? "automatic" : "manual",
     :metadata => {
       :order_id => '5278735C-1F40-407D-933A-286E463E72D8',
     }.merge(metadata || {}),
   )
 end
 
 def create_and_capture_payment_intent(amount, source_id, payment_method_id, customer_id = nil,
```
Comment (mshafrir-stripe) [inline web.rb]: Should we just get rid of the `create_and_capture_payment_intent` method and only have `create_payment_intent`? Then the calling methods can choose whether to call `confirm` or not.

File: web.rb [hunk 2]
```diff
@@ -166,15 +186,18 @@ def create_payment_intent(amount, source_id, payment_method_id, customer_id = ni
     :payment_method_types => ['card'],
     :description => "Example PaymentIntent",
     :shipping => shipping,
+    :return_url => return_url,
+    :confirm => return_url != nil,
```
Comment (mshafrir-stripe) [inline web.rb]: Is it always the case that if we have a `return_url` that we want to immediately confirm? They seem unrelated to me.

File: web.rb (hunk 1 above)
Comment (csabol-stripe) [reply web.rb]: I'll keep it for now as a convenience but we also have that option

File: web.rb (hunk 2 above)
Comment (csabol-stripe) [reply web.rb]: removed
